src/replaceResolverContext.cpp

```diff
@@ -1,14 +1,16 @@
 // Copyright 2019 Rodeo FX.  All rights reserved.
 #include "replaceResolverContext.h"
 
+#include "boost_include_wrapper.h"
+
 #include <pxr/pxr.h>
 #include <pxr/base/tf/diagnostic.h>
 #include <pxr/base/tf/hash.h>
 #include <pxr/base/tf/ostreamMethods.h>
 #include <pxr/base/tf/pathUtils.h>
 #include <pxr/base/tf/stringUtils.h>
 
-#include <boost/functional/hash.hpp>
+#include BOOST_INCLUDE(functional/hash.hpp)
```

Comment:
First time I'm seeing such include way. Are you running a cmake configure process on the file at some point?

Reply:
Ok so this `BOOST_INCLUDE`  is actually a macro defined here https://github.com/rodeofx/rdo_replace_resolver/pull/4/files#diff-832ea8cd54dd1f877a23891aaf3ddd7d 

All good since we need it because of Houdini hboost library renaming not only symbols but also boost sources...